### PR TITLE
deprecate django secure

### PR DESCRIPTION
**Pull request by aahunter**

### CODE REVIEW

**Comment by aahunter** [comment]
I think this breaks admin-pane.

File: yabi/yabi/settings.py [hunk 1]
```diff
@@ -33,14 +33,13 @@
 # set debug, see: https://docs.djangoproject.com/en/dev/ref/settings/#debug
 DEBUG = env.get("debug", not PRODUCTION)
 
-# django-secure
+# https://docs.djangoproject.com/en/1.8/ref/middleware/#django.middleware.security.SecurityMiddleware
 SECURE_SSL_REDIRECT = env.get("secure_ssl_redirect", PRODUCTION)
-# We do clickjacking support using Django's middleware. See MIDDLEWARE_CLASSES
-SECURE_FRAME_DENY = False
+SECURE_SSL_HOST = env.get("secure_ssl_host", False)
```
Comment (sztamas) [inline yabi/yabi/settings.py]: This should be a hostname not a boolean, so the False default doesn't look right.

File: yabi/yabi/settings.py [hunk 2]
```diff
@@ -49,14 +48,15 @@
...
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.admindocs.middleware.XViewMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware'
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'django.middleware.security.SecurityMiddleware',
```
Comment (sztamas) [inline yabi/yabi/settings.py]: I would put the SecurityMiddleware as first.

Comment (aahunter) [inline yabi/yabi/settings.py]: Checking docs, you're correct: https://docs.djangoproject.com/en/1.8/topics/http/middleware/

File: yabi/yabi/settings.py (hunk 1 above)
Comment (aahunter) [inline yabi/yabi/settings.py]: From the docs, you are correct: https://docs.djangoproject.com/en/1.8/ref/settings/#secure-ssl-host

**Comment by aahunter** [comment]
That secure_ssl_host change breaks Django.

django.core.exceptions.ImproperlyConfigured: Set the secure_ssl_host env variable

**Pull request merged**